lib/runtime_tools/src/dbg.erl

```diff
@@ -2375,6 +2375,20 @@ do_relay_1(RelP, Session) ->
 	    do_relay_1(RelP, Session)
     end.
 
+-doc false.
+dhandler_default_out() ->
+    %% When user (human) sets up dbg over remsh, group_leader is on other node.
+    %% In this case, pass printed info to the remote group_leader
+    %% so the user (human) can see what's happening
+    case node(group_leader()) == node() of
```

Comment:
Could you add a check here, that looks for if the current process is a shell.
`{_Key, Ancestors} = process_info(self(),{dictionary, '$ancestors'})), shell:whereis() == hd(Ancestors).`

Reply:
done